requirements-dev.txt

```diff
@@ -0,0 +1,3 @@
+flake8
+autopep8
```

Comment:
Any objections against using black instead?

Reply:
black is ok

Reply:
Done here
https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/46099e910dd2c9fcc4dc28e13003b3087d70a0a6